tasks/request_from_archive/request_from_archive.py

```diff
@@ -593,6 +610,39 @@ def restore_object(
     )
 
 
+def set_optional_event_property(event: Dict[str, Any], source_path: str, target_path: str,
+                                logger: logging.Logger) -> None:
```

Comment:
Why are we passing a `logger: logging.Logger` value here when we have already defined a global `LOGGER` object that we can use? Recommend you use the global LOGGER object instead of passing a value here.

Reply:
This was my change. I'm okay with switching this, as once we go clean this will be passed in via use-case constructor and placed as a class property.

Reply:
will use the existing LOGGER

Reply:
updated to use existing logger